Cargo.toml

```diff
@@ -21,15 +21,19 @@ travis-ci = { repository = "jonhoo/fantoccini" }
 maintenance = { status = "looking-for-maintainer" }
 
 [dependencies]
-webdriver = "0.39.0"
-url = "1.5.0"
+webdriver = "0.39"
+url = "1.5"
 serde = "1.0"
 serde_json = "1.0"
-futures = "0.1.14"
+futures = "0.1"
```

Comment:
It's not quite true that this is a "meaningless change". The patch version _is_ taken into considering by cargo, in that it will never pull a version _older_ than the given version. This may be important for dependencies that have max-version constraints, and can also be used to indicate required features from new patch versions so that cargo knows it has to fetch a version newer than what's in the user's `Cargo.lock`.